packages/app-desktop/gui/NoteEditor/NoteBody/TinyMCE/TinyMCE.tsx

```diff
@@ -738,6 +738,24 @@ const TinyMCE = (props: NoteBodyEditorProps, ref: any) => {
 					code: { inline: 'code', remove: 'all', attributes: { spellcheck: 'false' } },
 					forecolor: { inline: 'span', styles: { color: '%value' } },
 				},
+				text_patterns: props.enableTextPatterns ? [
```

Comment:
Note: When enabled, this uses the default value (based on the upstream docs) **and** 1-2 Joplin-specific patterns (highlight, code).